calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -38,7 +38,32 @@ The geeky details of what you get:
   details='Policy:Calico,IPAM:AWS,CNI:AWS,Overlay:No,Routing:VPC Native,Datastore:Kubernetes'
 />
 
-To enable {{prodname}} network policy enforcement on an EKS cluster using the AWS VPC CNI plugin, follow these step-by-step instructions: [Installing Calico on Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/calico.html)
+First, create an Amazon EKS cluster.
+```bash
+eksctl create cluster --name  my-calico-cluster
+```
+
+Install the operator.
```

Comment:
A small detail, but please add spaces between text and other elements, such as code blocks.